src/DbPostgreSQLAdapter/Transaction.cpp

```diff
@@ -0,0 +1,26 @@
+#include "stdafx.h"
+#include "Transaction.h"
+
+#include "DbAdapterInterface/IDatabase.h"
+
+namespace systelab::db::postgresql {
+
+	Transaction::Transaction(IDatabase& database)
+		: m_database(database)
+	{
+		std::string operation = "BEGIN TRANSACTION";
```

Comment:
All these can be inlined, no need for the variable

Reply:
Yes, it does not add anything, I'll change it